client/app/billing/autoRenew/bulk/translations/Messages_fr_FR.json

```
@@ -0,0 +1,16 @@
+{
+  "billing_autorenew_bulk": "Vous avez sélectionné {{ count }} services",
+  "billing_autorenew_bulk_cancel": "Annuler",
+  "billing_autorenew_bulk_confirm": "Valider",
+  "billing_autorenew_bulk_unavailable_pending_debt": "Les services ayant une facture en attente de réglement",
+  "billing_autorenew_bulk_unavailable_already_automatic": "Les services bénéficiant déjà du paiement automatique",
+  "billing_autorenew_bulk_unavailable_one_shot_automatic": "Les services ne pouvant pas bénéficier du paiement automatique",
+  "billing_autorenew_bulk_unavailable_one_shot_manual": "Les services non compatible avec le renouvellement manuel ",
```

```suggestion
  "billing_autorenew_bulk_unavailable_one_shot_manual": "Les services ne pouvant bénéficier du le renouvellement manuel ",
```
?

To have the same vocabulary everywhere